### ISSUE TITLE
Homepage does not render as expected on mobile

### ISSUE THREAD

**Issue by chartgerink** [STAPLE-verse/STAPLE#285]
Currently the homepage renders in a way that is not expected on mobile devices.

I do not 100% know whether the project aims to support mobile devices, so not sure whether you consider it an issue. In general, it is a recommended practice to ensure websites work with mobile devices these days, given that many people end up using it this way.

Wanted to flag this in case you are not yet aware of this.
![IMG_0357](https://github.com/user-attachments/assets/73fcdd98-0ef7-4cbe-83f5-885e30ab7e4a)

**Comment by doomlab** [comment]
Yes, I noticed this last night ... I'm not sure if it's the column div rendering or what? 

I know the rest of the site is somewhat awkward on mobile, so I think our (longer) term plan was to create a separate mobile version for smaller screens that was significantly paired down. Do you see what's causing the issue on this page though? I will also investigate (and can probably leave a note on the page about future plans so people are aware).

**Comment by chartgerink** [comment]
It's the combination of the `flex` and `justify-content` at the moment. Generally, the CSS may need a good solid pass to ensure things display as expected (if there are articulated expectations).

Honestly, I get that you say you want to create a separate mobile version and I bet this will lead to a frustrating UX for folk. It is not an unreasonable expectation that things work this way and you cannot prevent it. Maybe not for end of October, but for the first complete version 👍
